docs/Schema.sql

```diff
@@ -0,0 +1,61 @@
+ /*
+    The tables are in the order they will need to be created in our code dude to references.
+ */
+
+
+ /*
+    This is table on each university in the United States. study_set and user_info will 
+    link to it because it is better than having all the information about a university 
+    in multiple tables
+*/
+CREATE TABLE university(
+    id INT PRIMARY KEY,
+    name TEXT,
+    state TEXT,
+);
+
+/*
+    This table holds the relevant information for each user. university_id  holds the 
+    id of the university the user is currently enrolled in. The study_set_id array 
+    contains the id of all study_set created by the user.
+*/
+CREATE TABLE user_info(
+   id INT,
+   user_name TEXT,
+   full_name TEXT,
+   email TEXT,
+   verified BOOLEAN,
+   university_id INT REFERENCES university(id),
+   study_set_id INT [],
```

Comment:
Delete this field, each study set already has a reference to the owner ID. You typically follow relationships from child to parent.